<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The LETHIMCOOK SCANNER</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QLabel" name="imglabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>511</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>Img</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>511</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="progresslayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Contours">
         <property name="text">
          <string>Take a photo or upload an image to scan for patties</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="data">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>160</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="c1label">
       <property name="text">
        <string></string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="c2label">
       <property name="text">
        <string></string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="c3_label">
       <property name="text">
        <string></string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="c4_label">
       <property name="text">
        <string></string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="c5_label">
       <property name="text">
        <string></string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="c1_value">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="c2_value">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="c3_value">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="c4_value">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="c5_value">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ComponentLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-- Add to SIM --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ComponentValue">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Click to Add to Sim</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cameralabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>431</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>Click the 'Start Camera' Button to Begin Video</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>431</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="beginButton">
         <property name="text">
          <string>Start Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop Camera</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="selectCam">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="photoButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Take photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UploadImageButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Upload Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="AddSimulatorButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>429</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Manual Control Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>